src/types/excel.ts

```diff
@@ -0,0 +1,5 @@
+export interface HeaderItemProps {
```

Comment:
The name should be HeaderItem, not a suffix with Props, cause it is not a component.